limits/util.py

```diff
@@ -6,9 +6,12 @@
 
 from .limits import GRANULARITIES
 
-
+SINGLE_EXPR = re.compile(
```

Comment:
It's used as a compiled expression on line 44.
```python
amount, _, multiples, granularity_string = SINGLE_EXPR.match(limit).groups()
```
I can change it of course if you want that.